optuna/integration/skopt.py

```diff
@@ -243,15 +243,7 @@ def __init__(
 
         dimensions = []
         for name, distribution in sorted(self._search_space.items()):
-            if isinstance(distribution, distributions.UniformDistribution):
-                # Convert the upper bound from exclusive (optuna) to inclusive (skopt).
```

Comment:
Sorry after being merged the original PR but could you please copy this comment under `FloatDistribution`?

Reply:
Done in 5d75afc0630dc3d088151cfb4b6a09eba848316a.